ledger/participant-integration-api/src/main/scala/platform/store/backend/common/TransactionPointwiseQueries.scala

```diff
@@ -41,11 +41,6 @@ class TransactionPointwiseQueries(
             t.event_sequential_id_last
          FROM
             participant_transaction_meta t
-         JOIN parameters p
```

Comment:
join on the parameters table not needed anymore.
also previously we had performance issues with this kind of joins (...on a frequently changing table).